src/automation/azext_automation/aaz/latest/automation/hybrid_runbook_worker_group/_create.py

```diff
@@ -0,0 +1,246 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation hybrid-runbook-worker-group create",
+)
+class Create(AAZCommand):
+    """Create a hybrid runbook worker group.
+    """
+
+    _aaz_info = {
+        "version": "2022-02-22",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.automation/automationaccounts/{}/hybridrunbookworkergroups/{}", "2022-02-22"],
+        ]
+    }
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        self._execute_operations()
+        return self._output()
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.automation_account_name = AAZStrArg(
+            options=["--automation-account-name"],
+            help="The name of the automation account.",
+            required=True,
+            id_part="name",
+        )
+        _args_schema.hybrid_runbook_worker_group_name = AAZStrArg(
+            options=["-n", "--name", "--hybrid-runbook-worker-group-name"],
+            help="The hybrid runbook worker group name",
+            required=True,
+            id_part="child_name_1",
+        )
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+
+        # define Arg Group "Properties"
+
+        _args_schema = cls._args_schema
+        _args_schema.credential = AAZObjectArg(
+            options=["--credential"],
+            arg_group="Properties",
+            help="Sets the credential of a worker group.",
```

Comment:
We need to use the first person voice in the help message
```suggestion
            help="Set the credential of a worker group.",
```